aliyun-ecs/src/test/java/org/jclouds/aliyun/ecs/compute/ECSComputeServiceLiveTest.java

```diff
@@ -16,98 +16,78 @@
-            new Statement[] { AdminAccess.standard(), Statements.exec("sleep 50"), InstallJDK.fromOpenJDK() }));
-      return template;
-   }
+   protected void createAndRunAServiceInGroup(String group) throws RunNodesException {
+      // note that some cloud providers do not support mixed case tag names
+      ImmutableMap<String, String> userMetadata = ImmutableMap.of("test", group);
 
-   @Override
-   protected Module getSshModule() {
-      return new SshjSshClientModule();
-   }
+      ImmutableSet<String> tags = ImmutableSet.of(group);
+      Stopwatch watch = Stopwatch.createStarted();
 
-   @Override
-   @Test(expectedExceptions = AuthorizationException.class)
-   public void testCorrectAuthException() throws Exception {
-      ComputeServiceContext context = null;
-      try {
-         Properties overrides = setupProperties();
-         overrides.setProperty(provider + ".identity", "MOM:MA");
-         overrides.setProperty(provider + ".credential", "MIA");
-         context = newBuilder().modules(ImmutableSet.of(getLoggingModule(), credentialStoreModule)).overrides(overrides)
-               .build(ComputeServiceContext.class);
-         // replace listNodes with listImages as it doesn't require `projectId`
-         context.getComputeService().listImages();
-      } catch (AuthorizationException e) {
-         throw e;
-      } catch (RuntimeException e) {
-         e.printStackTrace();
-         throw e;
-      } finally {
-         if (context != null)
-            context.close();
-      }
-   }
+      template = buildTemplate(templateBuilder());
+      template.getOptions().inboundPorts(22, 8080).blockOnPort(22, 300).userMetadata(userMetadata).tags(tags);
 
-   @Override
-   public void testOptionToNotBlock() throws Exception {
-      // Aliyun ECS ComputeService implementation has to block until the node
-      // is provisioned, to be able to return it.
-   }
+      NodeMetadata node = getOnlyElement(client.createNodesInGroup(group, 1, template));
+      long createSeconds = watch.elapsed(TimeUnit.SECONDS);
 
-   @Override
-   protected void checkUserMetadataContains(NodeMetadata node, ImmutableMap<String, String> userMetadata) {
-      // The ECS API does not return the user data
+      final String nodeId = node.getId();
+
+      checkUserMetadataContains(node, userMetadata);
+      checkTagsInNodeEquals(node, tags);
+
+      getAnonymousLogger().info(
+              format("<< available node(%s) os(%s) in %ss", node.getId(), node.getOperatingSystem(), createSeconds));
+
+      watch.reset().start();
+
+      client.runScriptOnNode(nodeId, AdminAccess.builder().adminUsername("web").build(), nameTask("admin-web"));
+
+      long configureSeconds = watch.elapsed(TimeUnit.SECONDS);
+
+      getAnonymousLogger().info(
+              format(
+                      "<< configured node(%s) in %ss",
+                      nodeId,
+                      configureSeconds));
```

Comment:
This is not really honoring the purpose of the method, as it is not creating a service. What is the real problem with the base test? Jetty? Can we then change and run a different service? Something like a `python -m SimpleHTTPServer`?
I don't like this kind of overrides in tests. They alter the jclouds ComputeService impl contract, and it is unlikely we are revisiting this file once we fix some stuff in core/compute.

Reply:
I actually like the idea of using `python -m SimpleHttpServer` instead of installing `java` and `jetty` as it takes much longer

If we agree on this I think i can open a PR for jclouds/jclouds and change the ComputeServiceLiveTest for all of the providers

Reply:
Ill remove the override as I think we can accept to have this test broken until we fix the BaseComputeServiceLiveTest